skimage/morphology/heap_general.pxi

```diff
@@ -8,6 +8,8 @@ Copyright (c) 2009-2011 Broad Institute
 All rights reserved.
 
 Original author: Lee Kamentsky
+
+2018.10: Mark Harfouche: release the GIL
```

Comment:
We've decided to keep all copyright and author information outside of files, and in `CONTRIBUTORS.txt`.  So you are very welcome to add your name there.  We prefer not to modify these files, because it makes it hard to track licenses, and also cause headaches for our Debian and other packagers, who each time have to update individual file licenses.

Reply:
Ok,  I didn't know if you wanted to track the modifications to vendored files in the files themselves.